app/javascript/src/dashboard.js

```diff
@@ -1,4 +1,14 @@
 /* global $ */
+var defineCaseContactsTable  = function () {
```

Comment:
in order to test this behavior, I had to pull it out of the `$('document').ready(...)`*.

*https://stackoverflow.com/questions/29153733/how-to-unit-test-a-document-ready-function-using-jasmine